src/import-service/serverless.yml

```diff
@@ -34,6 +34,56 @@ resources:
       Type: AWS::SQS::Queue
       Properties:
         QueueName: catalogItemsQueue
+    GatewayResponseDefault400:
+      Type: AWS::ApiGateway::GatewayResponse
+      Properties:
+        ResponseParameters:
+          gatewayresponse.header.Access-Control-Allow-Origin: "'*'"
+          gatewayresponse.header.Access-Control-Allow-Headers: "'*'"
+        ResponseType: DEFAULT_4XX
```

Comment:
The `DEFAULT_4XX` response type already covers `ACCESS_DENIED` and `UNAUTHORIZED` response types